lib/elixir/scripts/path_helpers.exs

```diff
@@ -0,0 +1,78 @@
+# SPDX-License-Identifier: Apache-2.0
+# SPDX-FileCopyrightText: 2021 The Elixir Team
+# SPDX-FileCopyrightText: 2012 Plataformatec
+
+# Beam files compiled on demand
+path = Path.expand("../tmp/beams", __DIR__)
+File.rm_rf!(path)
+File.mkdir_p!(path)
+Code.prepend_path(path)
+
+defmodule PathHelpers do
+  def fixture_path() do
+    Path.expand("../test/elixir/fixtures", __DIR__)
```

Comment:
This would be leaking Elixir fixtures to other test suites.... I am not sure we want to do that.